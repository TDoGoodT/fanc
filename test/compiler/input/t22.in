void main(){
    int x = 10;
    int y = 10;
    while(true) {
        print("__x__");
        printi(x);
        print("__y__");
        printi(y);
        while (y > 0) {
            if (y < 5) {
                break;
            } else {
                y = y - 2;
            }
        }
        x = x - 1;
        if (x == 0) {
            break;
        }
        continue;
        x = 100;
    }
}