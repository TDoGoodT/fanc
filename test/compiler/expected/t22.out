__x__
10
__y__
10
__x__
9
__y__
4
__x__
8
__y__
4
__x__
7
__y__
4
__x__
6
__y__
4
__x__
5
__y__
4
__x__
4
__y__
4
__x__
3
__y__
4
__x__
2
__y__
4
__x__
1
__y__
4
